build.gradle

```diff
@@ -77,14 +85,15 @@ minecraft {
 }
 
 repositories {
-    if (System.env.USE_SUBMODULES) {
+    if (!System.env.RELEASE) {
```

Comment:
The switch is used when you're using submodules but if you just want to build without cloning submodules, it won't work